### PR TITLE
Notch updates

### PR DESCRIPTION
**Pull request by avicks**
- Update the camera preview and overlay UI to account for iOS devices with the notch. Use `safeAreaInsets` where possible.

## Pivotal Tracker tickets
_(link to all relevant Pivotal Tracker tickets)_

## Screenshots
_(if the changes are visual, provide screenshots with brief descriptions)_

### CODE REVIEW

**Comment by clutter-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Commit title 68512ff63b7e9709d106b03a90023fbd16693535 is 64 characters long (title should be limited to 50 characters).</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against 68512ff63b7e9709d106b03a90023fbd16693535
</p>

File: Sources/TiltUp/Screens/Camera/Views/CameraOverlayView.swift [hunk 1]
```diff
@@ -245,7 +245,7 @@ final class CameraOverlayView: UIView {
         return outerStackView
     }()
 
-    private lazy var previewImageView: UIImageView = {
+    public lazy var previewImageView: UIImageView = {
```
Comment (litso) [inline Sources/TiltUp/Screens/Camera/Views/CameraOverlayView.swift:248]: is this change needed?

**Review by litso — APPROVED** [review]
(Sorry for the delay in reviewing, usually we try to turn around a pr in less than 2 days)
Nice cleanup Alex, just one comment.
@jbrophy17 can show you how to release a new version of TiltUp when you are ready.

File: Sources/TiltUp/Screens/Camera/Views/CameraOverlayView.swift (hunk 1 above)
Comment (avicks) [reply Sources/TiltUp/Screens/Camera/Views/CameraOverlayView.swift:248]: Yes, its now used to set the previewView frame in the CameraController
